### ISSUE TITLE
[BUG] Attached crash report from app crashing 

### ISSUE THREAD

**Issue by exquisiteskink** [vitorpamplona/amethyst#374]
**Describe the bug**
App constantly crashing

```
type: crash
osVersion: google/raven/raven:13/TQ2A.230405.003.E1/2023041100:user/release-keys
package: com.vitorpamplona.amethyst:123
process: com.vitorpamplona.amethyst
processUptime: 74385 + 203 ms

java.lang.IllegalStateException: Required value was null.
	at androidx.compose.ui.node.DelegatableNodeKt.requireOwner(DelegatableNode.kt:345)
	at androidx.compose.ui.node.SemanticsModifierNodeKt.invalidateSemantics(SemanticsModifierNode.kt:41)
	at androidx.compose.foundation.text.modifiers.TextAnnotatedStringNode.doInvalidations(TextAnnotatedStringNode.kt:189)
	at androidx.compose.foundation.text.modifiers.TextAnnotatedStringElement.update(TextAnnotatedStringElement.kt:64)
	at androidx.compose.foundation.text.modifiers.TextAnnotatedStringElement.update(TextAnnotatedStringElement.kt:35)
	at androidx.compose.ui.node.NodeChainKt.updateUnsafe(NodeChain.kt:729)
	at androidx.compose.ui.node.NodeChainKt.access$updateUnsafe(NodeChain.kt:1)
	at androidx.compose.ui.node.NodeChain.updateNodeAndReplaceIfNeeded(NodeChain.kt:558)
	at androidx.compose.ui.node.NodeChain.updateFrom$ui_release(NodeChain.kt:130)
	at androidx.compose.ui.node.LayoutNode.setModifier(LayoutNode.kt:858)
	at androidx.compose.ui.node.ComposeUiNode$Companion$SetModifier$1.invoke(ComposeUiNode.kt:44)
	at androidx.compose.ui.node.ComposeUiNode$Companion$SetModifier$1.invoke(ComposeUiNode.kt:44)
	at androidx.compose.runtime.ComposerImpl$apply$operation$1.invoke(Composer.kt:1711)
	at androidx.compose.runtime.ComposerImpl$apply$operation$1.invoke(Composer.kt:1709)
	at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:814)
	at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:846)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:592)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:510)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(AndroidUiFrameClock.android.kt:34)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performFrameDispatch(AndroidUiDispatcher.android.kt:109)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performFrameDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(AndroidUiDispatcher.android.kt:69)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1229)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
	at android.view.Choreographer.doCallbacks(Choreographer.java:899)
	at android.view.Choreographer.doFrame(Choreographer.java:827)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7891)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@42d674a, androidx.compose.ui.platform.MotionDurationScaleImpl@a09b6bb, StandaloneCoroutine{Cancelling}@b71f7d8, AndroidUiDispatcher@21a8731]

```

**Comment by vitorpamplona** [comment]
This bug is killing us and I don't know where it comes from... :(

**Comment by bloodyeyesblackblood** [comment]
i don't know anything about it, but wouldn't it be better if android version and some more details would be included?
i don't had any crashes **yet**, so maybe something with a newer older software that can't really communicate with Java/Kotlin?

also this one looks similar but isn't really [(something that has been fixed with androidx.compose.ui:ui:1.4.0-alpha05) ](https://issuetracker.google.com/issues/258907850?pli=1)
 maybe this could help?

Android 11
MIUI 12.5.2

**Comment by vitorpamplona** [comment]
Seems to have been solved in v0.40.3.
